tests/functional/test_zilien_reader.py

```diff
@@ -206,10 +205,15 @@ def test_read(cls_or_technique, expected_series):
     if isinstance(cls_or_technique, str):
         expected_measurement_class = TECHNIQUE_NAME_TO_CLASS[cls_or_technique]
         measurement = Measurement.read(
-            PATH_TO_DATAFILE, reader="zilien", technique=cls_or_technique
+            PATH_TO_DATAFILE,
```

Comment:
This doesn't seem like the most elegant way to do it... 
what do we want the behavior to be when calling `ECMeasurement.read(path_to_tsv)" on a .tsv with associated mass scans? 
I guess in that case it should ignore the mass scans, right? I'll make the suggestion in my PR.